source/users.rst

```diff
@@ -15,10 +36,49 @@ PostgreSQL Operator comes with the following users:
       - Superuser, Create role, Create DB, Replication, Bypass RLS
     * - ``primaryuser``
       - Replication
-    * - ``testuser``
-      - 
+    * - ``pguser``
+      - Non-privileged user
+    * - ``pgbouncer``
+      - Еhe administrative user for the `pgBouncer connection pooler <http://pgbouncer.github.io/>`_
 
 The ``postgres`` user will be the admin user for the database instance. The
 ``primaryuser`` is used for replication between primary and replicas. The
-``testuser`` is a normal user that has access to the database “userdb” that is
-created for testing purposes.
+``pguser`` is the default non-privileged user.
+
+YAML Object Format
+******************
+
+The default name of the Secrets object for these users is
+``cluster1-users-secrets`` and can be set in the CR for your cluster in
+``spec.secretName`` to something different. When you create the object yourself,
+it should match the following simple format:
+
+.. code:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: my-cluster-secrets
+   type: Opaque
+   stringData:
+     pgbouncer: pgbouncer_password
+     postgres: postgres_password
+     primaryuser: primaryuser_password
+     pguser: pguser_password
```

Comment:
as per the spec it should be `user`, not `pguser`
@nonemax what is the reality?

Reply:
well, we have pguser in https://github.com/percona/percona-postgresql-operator/blob/main/deploy/users-secret.yaml

Reply:
we have it in default [cr](https://github.com/percona/percona-postgresql-operator/blob/main/deploy/cr.yaml#L25,) so thats why it `pguser` here